source/renderer/app/utils/assets.js

```diff
@@ -28,20 +24,19 @@ export const filterAssets = (
 
 /**
  *
- * This function receives a list os Wallet Assets
- * which don't contain fingerprint nor metadata
- *
- * And a function which gathers this missing items
- * from the `details` in the Assets Store
+ * This function receives a list os Tokens (the assets included in a wallet or transaction)
```

Comment:
This comment is misleading / incorrect and seems to be copied from the function below. This function just receives a single token and returns the `AssetToken` from what I can see :)

Reply:
Nice catch! ✔️